root/work/edit_form.tt

```diff
@@ -105,9 +105,54 @@
   </div>
 
   <div class="documentation">
-    [%- iswc_bubble(link_entity(work)) -%]
+    <div class="bubble" id="name-bubble">
+      <p>
+        [% l('The {doc|name} is the work’s canonical title in the language in which it was originally written.',
+             { doc => { href => doc_link('Work#Name'), target => '_blank' } }) %]
+      </p>
+    </div>
+
+    <div class="bubble" id="comment-bubble">
+      <p>
+        [% l('The {doc|disambiguation} field helps users distinguish between similarly-named works.',
+             { doc => { href => doc_link('Disambiguation_Comment'), target => '_blank' } }) %]
+      </p>
+      <p>
+        [% l('Please leave it blank if there are enough other differences between the works that they are unlikely to be confused.') %]
+      </p>
+    </div>
 
     [%- type_bubble(form.field('type_id'), work_types) -%]
+
+    <div class="bubble" id="languages-bubble">
+      <p>
+        [% l('The {doc|lyrics languages} should list any languages used in a significant portion of the work’s lyrics (or “No lyrics” if it has none).',
```

Comment:
Should we explicitly use [No lyrics] with the brackets, to match the selection option?

Reply:
Whoops, yes. I've updated this locally -- thanks!